package.json

```
@@ -15,17 +15,18 @@
   "scripts": {
     "ct": "git-cz",
     "prebuild": "rimraf dist",
-    "build": "tsc --build",
+    "build": "tsc --build tsconfig.build.json",
     "build:w": "npm run build -- --watch",
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
+    "test:w": "npm run test -- --watch",
     "format": "prettier --write src/**/*",
     "format:check": "prettier --check src/**/*",
     "semantic-release": "semantic-release"
   },
   "dependencies": {
-    "@typescript-eslint/experimental-utils": "^2.19.2",
-    "eslint-plugin-sonarjs": "^0.5.0",
-    "tslib": "^1.10.0"
+    "@typescript-eslint/experimental-utils": "^2.34.0",
```

probably should update this to allow the recent v3 release
```suggestion
    "@typescript-eslint/experimental-utils": "^2.34.0 || ^3.0.0",
```